server/di/di.go

```diff
@@ -50,14 +50,16 @@ func NewDIContainer(client clientset.Interface, restclientCfg *restclient.Config
 	c.priorityClassService = priorityclass.NewPriorityClassService(client)
 
 	deleteServices := map[string]reset.DeleteService{
-		"node":                    c.nodeService,
+		"node":              c.nodeService,
+		"persistent volume": c.pvService,
+		"storage class":     c.storageClassService,
+		"priority class":    c.priorityClassService,
+	}
+	deleteWithNamespaceService := map[string]reset.DeleteServicesForNamespacedResources{
```

Comment:
Rename to `deleteForNamespacedResources`

Reply:
oops, thanks!